uoyweek.cpp

```diff
@@ -83,19 +101,19 @@ Term * getTerm(time_t now) {
 string getString(bool capitalize) {
     time_t now = time(0);
     Term *term = getTerm(now);
-	string code = term->code;
+    string code = term->code;
     int week = getWeek(now, term);
-    string day = days[gmtime(&now)->tm_wday].substr(0,3);
-	if (capitalize) {
-		code[0] = std::toupper(code[0]);
-		day[0] = std::toupper(day[0]);
-	}
-    int length = strlen(term->code.c_str()) + 2 + 3 + (week>=10?2:1) + 1;
-    char* str = (char*) malloc(sizeof(char) * length);
+    string day = DAYS[gmtime(&now)->tm_wday].substr(0,3);
+    if (capitalize) {
+        code[0] = std::toupper(code[0]);
+        day[0] = std::toupper(day[0]);
+    }
+    int length = strlen(term->code.c_str()) + 2 + 3 + (week/10 + 1) + 1;
+    char *str = (char *)malloc(sizeof(char) * length);
     snprintf(str, length, "%s/%d/%s", code.c_str(), week, day.c_str());
     return str;
 }
 
 int main(void) {
-    cout << getString(false) << endl;
+    std::cout << getString(false) << std::endl;
```

Comment:
Avoid endl unless you actually require a flush - '\n' suffices

Reply:
TIL. Is this alright since it's the only thing that's getting printed or still unnecessary?

Reply:
For this, it's indeed largely irrelevant, but can have a big impact - http://demin.ws/blog/english/2012/04/25/endl-vs-new-line/